filebeat/filebeat.reference.yml

```diff
@@ -507,6 +507,16 @@ filebeat.prospectors:
   # Maximum size of the message received over UDP
   #max_message_size: 10240
 
+#------------------------------ Tcp prospector --------------------------------
```

Comment:
s/Tcp/TCP

Reply:
+1 for that change, I've went with `Tcp` even if  golang style would complain because of existing code, I wanted consistency with our `Udp` input. I guess I can make a PR on top of `UDP` to be all caps. :D

Reply:
Yep, saw the UDP one when I reviewed one of your other PR's and was thinking the same.